### DISCUSSION TITLE
MP4 error

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#275] [General]
For every file that's already mp4 the converter generates an error that it can't be generated. 
Is that my installation?
If it's not my install, can the conversions for mp4 files be skipped so that they won't generate errors?
Or to receive one cumulative window with all conversion messages?

**Comment by sircharlo** [comment]
No you're absolutely right; I changed my filename matcher dependency (called a "globber") a few weeks ago due to some compatibility issues with the old one, but it turns out that the syntax to exclude certain filetypes (like MP4 files) changed a bit in the new dependency. As a result, all files are getting analyzed instead of just those that can actually be converted.

Will be resolved for next point release. In the meantime, it's not "mission critical", just annoying. Apologies!

**Reply by riemsesy** [reply]
aha... good to know it's known :-)
